src/auto-line-wipe/SingleAutoLineWipeSpell.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: © 2024 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {DssEmergencySpell} from "../DssEmergencySpell.sol";
+
+interface LineMomLike {
+    function wipe(bytes32 ilk) external returns (uint256);
+    function autoLine() external view returns (address);
+}
+
+interface AutoLineLike {
+    function ilks(bytes32 ilk)
+        external
+        view
+        returns (uint256 maxLine, uint256 gap, uint48 ttl, uint48 last, uint48 lastInc);
+}
+
+contract SingleAutoLineWipeSpell is DssEmergencySpell {
+    LineMomLike public immutable lineMom = LineMomLike(_log.getAddress("LINE_MOM"));
+    bytes32 public immutable ilk;
+
+    event Wipe(bytes32 indexed ilk, uint256 prevLine);
```

Comment:
`event Wipe` differs in signature from the one used in MultiAutoLineWipeSpell. `prevLine` can be easily fetched offchain, so removing it will also make the spell execution a bit cheaper (one less SLOAD).

Reply:
Actually, for this one, `ilk` is an immutable contract parameter, so it can also be easily fetched.

Reply:
Please check now.

Reply:
Still differs from the multi-auto-line (`Wipe(ilk)` vs `Wipe()`). But agree `Ilk` not necessary for the single wipe spell.

Reply:
It's the same pattern as the other spells in this repo.